sympy/solvers/ode/tests/test_systems.py

```diff
@@ -1210,6 +1210,81 @@ def test_sysode_linear_neq_order1_type4():
     assert checksysodesol(eqs8, sol8) == (True, [0, 0, 0, 0])
 
 
+def test_component_division():
```

Comment:
There should be direct tests for the `_component_division` function and the `eqs2dict` and `dict2graph` functions.